axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/QuerySerializer.java

```diff
@@ -56,6 +56,7 @@ public class QuerySerializer {
     private final GrpcObjectSerializer<Object> exceptionDetailsSerializer;
     private final GrpcPayloadSerializer payloadSerializer;
     private final GrpcMetadataSerializer metadataSerializer;
+    @SuppressWarnings("unused")
```

Comment:
I am skipping this file as it'll be removed/replaced as part of https://github.com/AxonFramework/AxonFramework/issues/3488.